src/credentials/FabricTable.cpp

```diff
@@ -806,10 +839,12 @@ CHIP_ERROR FabricTable::Init(PersistentStorageDelegate * storage)
     }
     mNextAvailableFabricIndex.SetValue(kMinValidFabricIndex);
 
+    ReturnErrorOnFailure(mLastKnownGoodTime.Init(storage));
```

Comment:
Can we make this *not fail* ? Otherwise failure to load LKG, which can revert to a good value with the certificates set on failure to load, could cause a bootloop of failure at init on a product, if anything goes awry with loading failsafe, somehow...

Reply:
Yes, and actually I need to make it not fail for other reasons.

Even though it is required that nodes keep  track of Last Known Good Time, it doesn't really make sense for them to pay the penalty for not doing this unless current time AND last known good time aren't known at the time of certificate validation.

For nodes that can guarantee they will always have current time, this will simplify implementation.

Reply:
Done.